src/Altinn.Platform/Altinn.Platform.Profile/Profile/Startup.cs

```diff
@@ -1,190 +1,9 @@
-using System;
-using System.IO;
-using System.Reflection;
-using Altinn.Common.AccessToken;
-using Altinn.Common.AccessToken.Configuration;
-using Altinn.Common.AccessToken.Services;
-using Altinn.Platform.Profile.Configuration;
-using Altinn.Platform.Profile.Filters;
-using Altinn.Platform.Profile.Health;
-using Altinn.Platform.Profile.Services.Implementation;
-using Altinn.Platform.Profile.Services.Interfaces;
-using Altinn.Platform.Telemetry;
-using AltinnCore.Authentication.JwtCookie;
-using Microsoft.ApplicationInsights.Channel;
-using Microsoft.ApplicationInsights.Extensibility;
-using Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel;
-using Microsoft.AspNetCore.Authorization;
-using Microsoft.AspNetCore.Builder;
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.AspNetCore.Http;
-using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Hosting;
-using Microsoft.Extensions.Logging;
-using Microsoft.IdentityModel.Logging;
-using Microsoft.IdentityModel.Tokens;
-using Microsoft.OpenApi.Models;
-using Swashbuckle.AspNetCore.SwaggerGen;
-
 namespace Altinn.Platform.Profile
 {
     /// <summary>
-    /// Profile startup
+    /// The Startup class has been kept as a common hook for tests in the test project.
```

Comment:
Program can be used as the common hook for the tests instead of keeping this file.

Reply:
Yeah, but I would have to change the protection level. Make it public. 

Technically any class or interface from the project would work. ErrorController, HealthCheck, IUserProfile. I even considered adding a new interface, ITestProjectAssemblyHookOrSomething. None of them feels like good options.

I kept the startup class because then I don't need to change the WebApplicationFactorySetup class in the test project. I'm now considering moving parts of Program back into startup. I could make WebApplicationFactorySetup generic.

Reply:
I made additional changes that made it possible to delete Startup.